org.jacoco.tests/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>../org.jacoco.ant.test</module>
     <module>../jacoco-maven-plugin.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../org.jacoco.tests.coverage</module>
```

Comment:
@marchof Actually the more I think about this story of exclusion of dependencies, the more I tend to think that `report-aggregate` can simply be an enhancement of `report` by ability to select dependencies.